specification/policyinsights/resource-manager/Microsoft.PolicyInsights/stable/2023-03-01/checkPolicyRestrictions.json

```diff
@@ -378,6 +409,41 @@
           "readOnly": true
         }
       }
+    },
+    "PolicyEffectDetails": {
+      "type": "object",
+      "description": "The details of the effect that was applied to the resource.",
+      "properties": {
+        "policyEffect": {
+          "description": "The effect that was applied to the resource. http://aka.ms/policyeffects",
+          "type": "string",
+          "readOnly": true
+        }
+      }
+    },
+    "PolicyEvaluationDetails": {
```

Comment:
This change introduced an duplicate schema issue. The definition is different from `stable/2019-10-01/policyStates.json#/definitions/PolicyEvaluationDetails`. Could you consolidating them or rename the model?